tools/docker-compose/ansible/roles/sources/templates/receptor-hop.conf.j2

```diff
@@ -10,3 +10,6 @@
 
 - tcp-listener:
     port: 5555
+
+- control-service:
+    service: control
```

Comment:
Does receptorctl really work on the hop node? Have you been able to get it to work?

Reply:
Yes, it does.  @fosterseth and I worked that out in a pairing session, to find out how we could get hop nodes to show up at all in the output of `receptorctl status`.  Since this PR is no longer relying on that behavior for auto-discovery this could technically be dropped, but I believe we had found that the installer and aap-qe does have this turned on for hops so we should probably keep it for consistency.